### PR TITLE
Add recipe: gsee

### PR DESCRIPTION
**Pull request by sjpfenninger**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sjpfenninger** [comment]
@conda-forge/help-python I think this is ready for review. I did not manage to get the recipe build toolchain running locally so I am not 100% certain that the results work, but it's a relatively straightforward package, and the CI jobs all look like things worked. Maybe also for @conda-forge/help-python-c since this includes a small Cython/C component.

File: recipes/gsee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.15
```
Comment (xhochy) [inline recipes/gsee/meta.yaml]: Does this compile against the `numpy` API? Then you should fix this to the smallest supported `numpy` version.
```suggestion
    - numpy 1.15.*
```

File: recipes/gsee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  run:
+    - python
+    - joblib >=0.12
```
Comment (xhochy) [inline recipes/gsee/meta.yaml]: In relation to the above comment, you probably need a pinning for `numpy` here:
```suggestion
    - {{ pin_compatible('numpy') }}
    - joblib >=0.12
```

File: recipes/gsee/meta.yaml (hunk 1 above)
Comment (sjpfenninger) [reply recipes/gsee/meta.yaml]: Thanks, I checked the conda-forge docs, and this plus your second suggestion below seems exactly right.

**Pull request merged**